scanner/updater/version/VULNERABILITY_BUNDLE_VERSION

```diff
@@ -0,0 +1,8 @@
+# This file lists ACS Scanner v4 vulnerability definition versions.
+# Each version is linked to a branch of the codebase that exports that specific version.
+# The scanner updaters for the associated branch are responsible for regularly updating the vulnerability data for that definition bundle.
+# The updated data is then uploaded to Google Storage.
+# Each vulnerability definition version serves the Scanner requesting it, and the definitions are imported to that specific Scanner.
+# This process ensures that each version's vulnerability data is accurately captured and maintained in accordance with its respective version.
+dev,master
+v1,release-4.5
```

Comment:
OK, per ADR this should be git references, so use things like `tag/<version>` or `heads/<branch>`.

Also, see my comment on using a similar script to parse a RELEASE TAG (e.g. 4.6) and then select the most appropriate tag based on what was cut recently. Example: If you specify `4.6,v1` in this file and the last tag for `4.6.x` is `4.6.2`, then it would use that to build the bundle.

Reply:
Nitpick: Can we NOT use commas? Whitespaces would be easier for humans to read.

Reply:
I applied the tag reference change.
I feel like commas are as easy as whitespaces for humans to read tho.